.github/workflows/sycl-post-commit.yml

```diff
@@ -120,6 +120,8 @@ jobs:
       target_devices: "level_zero:gpu"
       sycl_toolchain_archive: ${{ needs.build-win.outputs.artifact_archive_name }}
       cxx: icx
+      # https://github.com/intel/llvm/issues/18458
+      env: "{'LIT_FILTER_OUT':'std_array.cpp'}"
```

Comment:
lgtm, in the future we could consider adding a `oneapi` lit feature or something so we could detect it inside lit, for one test its not worth it tho.